src/api/posts.ts

```diff
@@ -0,0 +1,7 @@
+import { Post } from '../types/Post';
+import { client } from '../utils/fetchClient';
+
+export const getUserPosts = (userId: Post['userId']) => {
+  return client.get<Post[]>(`
+/posts?userId=${userId}`);
```

Comment:
The template literal for the URL is split across two lines. Consider keeping it on a single line for better readability and to avoid potential issues with whitespace.